### PR TITLE
iroha-186: schema updated

### PR DESCRIPTION
**Pull request by arndey**

### CODE REVIEW

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt [hunk 1]
```diff
@@ -33,25 +34,26 @@ import jp.co.soramitsu.iroha2.generated.datamodel.permissions.PermissionToken
...
-const val CAN_BURN_ASSET_WITH_DEFINITION = "can_burn_asset_with_definition"
-const val CAN_BURN_USER_ASSETS_TOKEN = "can_burn_user_assets"
-const val CAN_REGISTER_DOMAINS_TOKEN = "can_register_domains"
-const val CAN_TRANSFER_USER_ASSETS_TOKEN = "can_transfer_user_assets"
-const val CAN_UNREGISTER_ASSET_WITH_DEFINITION = "can_unregister_asset_with_definition"
-
-const val ACCOUNT_ID_TOKEN_PARAM_NAME = "account_id"
-const val ASSET_ID_TOKEN_PARAM_NAME = "asset_id"
-const val ASSET_DEFINITION_PARAM_NAME = "asset_definition_id"
+val CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN = "can_set_key_value_in_user_assets".asName()
```
Comment (rkharisov) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt:40]: Let's make all them lazy to not load all them in memory
```suggestion
val CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN = by lazy { "can_set_key_value_in_user_assets".asName() }
```

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt [hunk 2]
```diff
@@ -44,6 +45,7 @@ import jp.co.soramitsu.iroha2.generated.datamodel.events.pipeline.EventFilter as
 
 open class Iroha2Client(
     open var peerUrl: URL,
+    open var telemetryUrl: URL? = null,
```
Comment (rkharisov) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt:48]: `telemetryUrl` is nullable, if it is not set, then calling `status` will have strange behaviour. As far as I understand telemetry will be on the other port, but always on the same host with `peerUrl`. I think we can create `telemetryUrl` from `peerUrl`
```suggestion
open var telemetryUrl: URL = URL(peerUrl.protocol, peerUrl.host, DEFAULT_TELEMETRY_PORT, peerUrl.file),
```

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt [hunk 3]
```diff
@@ -235,28 +243,33 @@ open class Iroha2Client(
                 }
                 is TransactionRejectionReason.NotPermitted -> reason.notPermittedFail.reason
                 is TransactionRejectionReason.SignatureVerification -> reason.signatureVerificationFail.reason
-                is TransactionRejectionReason.UnexpectedGenesisAccountSignature -> "Genesis account can sign only transactions in the genesis block"
-                is TransactionRejectionReason.UnsatisfiedSignatureCondition -> reason.unsatisfiedSignatureConditionFail.reason
+                is TransactionRejectionReason.UnexpectedGenesisAccountSignature ->
+                    "Genesis account can sign only transactions in the genesis block"
+                is TransactionRejectionReason.UnsatisfiedSignatureCondition ->
+                    reason.unsatisfiedSignatureConditionFail.reason
+                else -> throw IllegalArgumentException(
```
Comment (rkharisov) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt:250]: The branch `WasmExecution` is not covered. Please cover it. With this branch `else` will be unnecessary

**Comment by rkharisov** [comment]
Good job!

**Review by Mingela — APPROVED** [review]
please resolve the comments before merging

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt [hunk 4]
```diff
@@ -44,6 +45,7 @@ import jp.co.soramitsu.iroha2.generated.datamodel.events.pipeline.EventFilter as
 
 open class Iroha2Client(
     open var peerUrl: URL,
+    open var telemetryUrl: URL? = URL(peerUrl.protocol, peerUrl.host, DEFAULT_TELEMETRY_PORT, peerUrl.file),
```
Comment (rkharisov) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt:48]: Please make it non null explicitly. It is possible to pass `null` as an arg

**Review by rkharisov — APPROVED** [review]

**Pull request merged**